src/modules/extendedCallWindow/i18n/sv_SE.root.json

```diff
@@ -0,0 +1,163 @@
+			"title": "Visa ARR-detaljer"
+		},
+		"arrTime": {
+			"description": "Visar vad som skulle vara den längsta restiden efter att ha valt denna ARR, om du stannar med musen kort över en ARR.",
+			"title": "Längsta restid"
+		},
+		"centerMap": {
+			"description": "Låter dig centrera kartan i huvudfönstret på användningsstället med en knapptryckning.",
+			"title": "Centrera karta"
+		},
+		"extendedMissingVehicles": {
+			"description": "Konverterar visning av ytterligare fordringar till en tydligare tabell.",
+			"title": "Förbättra fordon som behövs"
+		},
+		"generationDate": {
+			"description": "Visas bredvid adressen när uppdraget genererades.",
+			"title": "Generation time"
+		},
+		"hideVehicleList": {
+			"description": "Döljer normalt användarlistan och öppnar den sedan genom att trycka på en knapp.",
+			"title": "Döljer fordonslistan"
+		},
+		"loadMoreVehiclesInHeader": {
+			"description": "Visar knappen \"Ladda om fordon\", om tillgänglig, i rubrikfältet.",
+			"title": "Ladda om fordon i rubrik"
+		},
+		"missionKeywords": {
+			"color": "bakgrundsfärg",
+			"description": "Få ditt eget nyckelord visat bredvid uppdragsnamnet",
+			"keyword": "sökord",
+			"prepend": "Visa före uppdragsnamnet",
+			"preview": "Preview",
+			"title": "Uppdrag nyckelord",
+			"uppdrag": "uppdrag"
+		},
+		"patientSummary": {
+			"description": "Visar bredvid antalet patienter vilken räddningsutrustning de behöver i kombination.",
+			"title": "Patientöversikt"
+		},
+		"redBorder": {
+			"description": "Skisserar genereringstiden i rött om vadet går ut nästa natt.",
+			"title": "Röd kant"
+		},
+		"tailoredTabs": {
+			"color": "Färg",
+			"description": "Justera flikarna i infogningsfönstret här. \"Alla\" och \"Fortsätt varna\" kommer alltid att vara synliga,",
+			"vehicles": "fordonstyper",
+			"name": "name",
+			"title": "fordonskategorier"
+		},
+		"stickyHeader": {
+			"description": "Visar alltid det övre fältet i infogningsfönstret, även när du rullar nedåt.",
+			"title": "Visa alltid rubrikfält"
+		},
+		"yellowBorder": {
+			"description": "Gränsar genereringstiden gul om användningen är äldre än det angivna antalet timmar. Om siffran är 0 dras ingen kant.",
+			"title": "Gul kant"
+		}
+	},
+	"tailoredTabs": {
+		"defaultTabs": [
+			{
+				"name": "Brandbil",
+				"vehicleTypes": [
+					0,
+					1,
+					12
+				]
+			},
+			{
+				"name": "Övriga Brandbilar",
+				"vehicleTypes": [
+					2,
+					3,
+					4,
+					6,
+					7,
+					10,
+					11,
+					19,
+					21,
+					22,
+					23
+				]
+			},
+			{
+				"name": "Ambulans",
+				"vehicleTypes": [
+					5,
+					9
+				]
+			},
+			{
+				"name": "Polis",
+				"vehicleTypes": [
+					8,
+					13,
+					14,
+					15,
+					16,
+					17,
+					24
+				]
+			},
+			{
+				"name": "Brandfordon-Vatten",
+				"vehicleTypes": [
+					18,
+					20
+				]
+			},
+			{
+				"name": "Insatsstyrka",
+				"vehicleTypes": [
+					14,
+					17
+				]
+			}
```

Comment:
@Suchty112 please check if this is standard ingame or not :)

Reply:
It should be right.